.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,13 +4,13 @@
 
 medic/cht-core#[number]
 
-# Code review items
-
-- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
-- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
-- Tested: Unit and/or e2e where appropriate
-- Internationalised: All user facing text
-- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
+# Code review checklist
+<!-- Remove or comment out any items that do not apply to this PR; in the remaining boxes, replace the [ ] with [x]. -->
```

Comment:
Thanks @andy5995, this looks like a nice feature to take advantage of. If I understand correctly, the html comment syntax here will be seen when viewing it in an editor, but not when displayed in a browser (viewing a PR on GitHub). Is that correct? If so, this looks good to me. If the characters will render literally in the browser though, I'd like it to be modified to just be a regular line of text.

Reply:
@craig-landry I'm glad you like it. Yes, the comment syntax works here. To preview this, you can click on the "files changed" tab of the PR -> click the "more"(3 vertical dots) icon on the right -> view file.

Reply:
Nice, good stuff! Thanks @andy5995.